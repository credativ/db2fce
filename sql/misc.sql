-- misc tests
SELECT INT(5) - (INT(5) / 1000) * 1000 FROM SYSIBM.SYSDUMMY1;
SELECT 5 / 1000 FROM SYSIBM.SYSDUMMY1;
SELECT 5.0 / 1000 FROM SYSIBM.SYSDUMMY1;
--
SELECT * FROM SYSIBM.SYSDUMMY1 FETCH FIRST 1 ROWS ONLY;
