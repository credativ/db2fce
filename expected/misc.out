-- misc tests
SELECT INT(5) - (INT(5) / 1000) * 1000 FROM SYSIBM.SYSDUMMY1;
ERROR:  syntax error at or near "("
LINE 1: SELECT INT(5) - (INT(5) / 1000) * 1000 FROM SYSIBM.SYSDUMMY1...
                  ^
SELECT 5 / 1000 FROM SYSIBM.SYSDUMMY1;
 ?column? 
----------
        0
(1 row)

SELECT 5.0 / 1000 FROM SYSIBM.SYSDUMMY1;
        ?column?        
------------------------
 0.00500000000000000000
(1 row)

--
SELECT * FROM SYSIBM.SYSDUMMY1 FETCH FIRST 1 ROWS ONLY;
 ibmreqd 
---------
 X
(1 row)

